fedimint-core/src/module/mod.rs

```diff
@@ -578,6 +580,26 @@ pub trait CommonModuleInit: Debug + Sized {
     fn decoder() -> Decoder;
 }
 
+/// Can be used to send a notification to consensus server,
+/// to trigger immediate [`ServerModule::consensus_proposal`] call.
+///
+/// Useful for modules to optionally decrease latency of consensus
+/// item handling.
+#[derive(Debug, Clone)]
+pub struct ConsensusProposalTrigger(watch::Sender<u64>);
+
+impl From<watch::Sender<u64>> for ConsensusProposalTrigger {
+    fn from(value: watch::Sender<u64>) -> Self {
+        Self(value)
+    }
+}
+
+impl ConsensusProposalTrigger {
+    pub fn send(&self) {
+        self.0.send_modify(|v| *v = v.wrapping_add(1))
```

Comment:
*Technically*, there is a race condition that, after 2^64 sends, a recipient that did not get to execute in the meantime will not notice the notification :zany_face:

Reply:
The polling is best effort and even then consensus doesn't guarantee anything, so *technically* still ok. 🤣